controllers/user.js

```diff
@@ -1,15 +1,36 @@
 const User = require('../models/user');
+const Exercise = require('../models/workoutlog');
+
 
 function index(req, res, next) {
-    User.find({}, function(err, user) {
-      res.render('user/index', { user,
-      user: req.user });
-    });
-  }
+ const user = req.user 
+    User.find({_id:req.user._id}, function(err, user) {
+      console.log(user)
+       Exercise.find({}, function(err, exercise) {
+      console.log("exercise", exercise)
+      res.render('user/index', { user, exercise });
+   })
+  })
+  };
+
+  
+//create function
+function create(req, res) {
+  console.log(req.body)
+  const exercise = new Exercise(req.body)
+  exercise.save(function (err){
+      if(err)
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7828699)
```suggestion
      if(err)
```